util/hash.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+import "github.com/evergreen-ci/utility"
+
+// GetSHA256Hash returns the SHA256 hash of the given string.
+func GetSHA256Hash(s string) string {
+	hasher := utility.NewSHA256Hash()
+	hasher.Write([]byte(s))
```

Comment:
We should handle the error here

Reply:
Whoops, I meant to use `Add` actually. Changed it.

[sha256 doesn't actually return an error](https://cs.opensource.google/go/go/+/refs/tags/go1.20:src/crypto/sha256/sha256.go;l=179-201), it's just conforming to the `io.Writer` interface.